services/DashSystem/dash_api.py

```
@@ -45,43 +57,111 @@
+    except Exception as e:
+        dash_system = None
+        dash_init_status["state"] = "failed"
+        dash_init_status["ended_at"] = time.time()
+        dash_init_status["elapsed_seconds"] = (
+            dash_init_status["ended_at"]
+            - (dash_init_status["started_at"] or dash_init_status["ended_at"])
+        )
+        dash_init_status["last_step"] = "failed"
+        dash_init_status["error"] = str(e)
+
+        logger.error(f"[STARTUP] Failed to initialize DASHSystem: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)